### DISCUSSION TITLE
Summary count and sum value

### DISCUSSION THREAD

**Discussion by agalvan002** [prometheus/client_python#799] [Q&A]
Hi , 

Can anyone explain me what are the differences between count and sum value in a summary in add_metric function?

Thanks!

**Comment by csmarchbanks** [comment]
Hello, `count_value` is the number of observations that were done and `sum_value` is the sum of all of the observations that were done. You can read more about them in our [documentation about summaries and histograms](https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations).

As an example, if you are creating a summary from 3 observations, say 1.0, 1.5, 1.8, then count would be 3 and sum would be 4.3.
